### DISCUSSION TITLE
Making quarto book recognize tables from gtsummary, gt, flextable and add them to list of tables

### DISCUSSION THREAD

**Discussion by abduazizR** [quarto-dev/quarto-cli#9307] [Q&A] [answered]
### Description

Hi,

I have an issue with using the `lot: true` option with quarto books. This option seems to only recognize tables created inside .qmd (markdown tables), but it does not recognize tables generated by different table packages inside R code chunks. Is there a way to make it work so tables generated by those packages can appear on the list of tables page?

**Comment by mcanouil** [comment]
Could you share a small **self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

Note that your tables should be cross-referenceable.


---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by abduazizR** [reply, marked as answer]
Thanks for the quick reply. I found the solution by using tbl-cap.

**Comment by cderv** [comment]
For reference and later reader, this is documented here in the doc: https://quarto.org/docs/authoring/tables.html#cross-references
